BrowserKit/Sources/SiteImageView/SiteImageHandler.swift

```diff
@@ -15,12 +15,11 @@ public class DefaultSiteImageHandler: SiteImageHandler {
     private let urlHandler: FaviconURLHandler
     private let imageHandler: ImageHandler
 
-    private let serialQueue = DispatchQueue(label: "com.mozilla.DefaultSiteImageHandler")
-    private var _currentInFlightRequest: String?
-    private var currentInFlightRequest: String? {
-        get { return serialQueue.sync { _currentInFlightRequest } }
-        set { serialQueue.sync { _currentInFlightRequest = newValue } }
-    }
+    /// Right now, multiple `SiteImageView`s each have their own `DefaultSiteImageHandler`. Ideally they'd all share a reference to
+    /// the same `DefaultSiteImageHandler` so we could properly queue and throttle requests to get favicon URLs, download images,
+    /// etc. Since that's a large architectural change, for now lets use a static queue so we can prevent too many duplicate calls to remotely fetching
```

Comment:
I think these lines are causing SwiftLint length warnings?

Reply:
Fixed!

Sidenote... I set the xcode page guide to be at 125 (to match swiftlint), but comments preceded by `///` were not in a monospaced font! 🤦‍♀️  Finally figured out why I kept getting these warnings lol.